### DISCUSSION TITLE
如何在replitt上搭建的alist使用webdav？

### DISCUSSION THREAD

**Discussion by html5syt** [AlistGo/alist#4607] [Q&A]
我在replit搭建的alist中无法使用webdav功能，每次登录后都会显示“method not allowed”，即使用raidrive也是如此。看文档说要配置反代，但我根本找不到配置的地方，加上端口号又无法访问alist，我该怎么办？

**Comment by Toserk1024** [comment]
Replit实际上就是把一个项目监听的端口，给反向代理到80或者443端口，并不需要手动去配置。我也是用的replit搭建的，webdav使用一切正常

**Reply by Synchronized1** [reply]
请问是如何配置的,配置的一直连不上

**Reply by Toserk1024** [reply]
> 请问是如何配置的,配置的一直连不上

Replit马上要废了，我现在都用手机+frp搭建的。到明年年初，replit使用动态程序部署就要托管收费了，只有静态部署是免费的，那样我还不如用github page，抓紧另寻出路吧。

如果你单纯问webdav配置的话：
URL：http(s)://{domain}/dav
用户名和密码你跟你网页登录时一样的，
domain要填你自己的域名

**Reply by Synchronized1** [reply]
感谢解答.......明年replit废了啊.....😔
